config.json-dist

```diff
@@ -0,0 +1,12 @@
+{
+    "default": {
+        "normandy_url": "https://localhost:8000/api/v1",
+        "api_token": "",
+        "verify": false
+    },
+    "development": {
+        "normandy_url": "https://normandy.dev.mozaws.net/api/v1",
+        "api_token": "",
+        "verify": true
+    }
```

Comment:
This also gives you room in the future to have other global configuration, which is something that might be nice in the future.